packages/web-pkg/src/components/SideBar/Spaces/Details/SpaceDetails.vue

```diff
@@ -64,11 +66,20 @@
           <space-quota :space-quota="resource.spaceQuota" />
         </td>
       </tr>
+      <tr v-if="showSpaceId">
+        <th scope="col" class="oc-pr-s oc-font-semibold" v-text="$gettext('Space ID')" />
+        <td class="oc-flex oc-flex-middle">
+          <div class="oc-text-truncate" v-text="resource.id" />
+          <oc-button class="oc-ml-s" appearance="raw" size="small" @click="copySpaceIdToClipboard">
+            <oc-icon :name="copySpaceIdIcon" />
+          </oc-button>
```

Comment:
Please add a `aria-label` since this is a button without text. Also, we should probably add a tooltip as well, no? Our buttons without text usually have it.

Reply:
good point!